### PR TITLE
[content update] #743-package-management-r-renv

### PR DESCRIPTION
**Pull request by Fernando-Iscar**
Hi all,

Following the guidelines in issue #743, I've updated the building block concerning [Package Management for R: renv](https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/renv/).

## Overview of the changes:

- Updated metadata: more keywords & improved alias.
- Added *Learning Goals* and *Additional Resources* sections.
- Improved content: adding more media and tips to illustrate & improve explanations quality.
- Reorganize structure: separate section for other R dependency management tools.

## Note:

The table in line 114, is displayed skewed to the left. After some failed tries to center it, I wonder if it is a problem related to website rendering. What do you think @thierrylahaije ?

Otherwise, I guess we can leave it as it is.

### CODE REVIEW

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md [hunk 1]
```diff
@@ -91,33 +96,70 @@ renv::restore()
...
 
-## Why renv is better than groundhog
+### Groundhog
 
-`groundhog` is another R package that makes projects reproducible. However, it doesn't work by keeping records of packages versions necessary for each project, like `renv` does. Instead you have to manually load libraries with the option of giving a date on when the wanted version was available. Thus, `renv` has more advantages because it can load all necessary packages in one go, without having to manually input the dates certain versions were available on.
+`groundhog` is another R package that facilitates project reproducibility. Unlike `renv` which maintains records of package versions specific to each project, `groundhog` requires users to manually load libraries by specifying a date to fetch the version available at that time.
 
+This can be challenging, as users have to recall the specific dates when certain versions were available. On the other hand, `renv` conveniently loads all necessary packages in one go, making the overall process smoother.
 
+## Additional Resources
```
Comment (srosh2000) [inline content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md:161]: Good to add a "summary" section with key bullets after the additional resources section

**Comment by lachlandeer** [comment]
@Fernando-Iscar --can you let me know when you've checked over roshini's points?

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
 author: "Ana Bianca Luca"
 authorlink: "https://www.linkedin.com/in/ana-bianca-luca-b555561b2/"
 aliases:
-  - /manage/packages
+  - /manage/R-packages
   - /use/renv
   - /replace/packrat
 ---
 
+## Learning Goals
```
[the remark this answers is not in the document]
Comment (Fernando-Iscar) [reply content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md]: Good observation!

✅I've changed the header from *Learning Goals* to *Overview*

✅I've made a short introduction before enlisting the learning goals

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md [hunk 3]
```diff
@@ -91,33 +96,70 @@ renv::restore()
...
+| `renv/activate.R`  | Activation script run by `.Rprofile`                   |
+| `renv/library`     | Private project library                                |
+| `renv/settings.json`| Project settings                    
+
+{{% tip %}}
+**Another Project in Rstudio?**
+
+When creating a new project in RStudio, you have the option to directly integrate `renv`.
+
+<p align = "center">
```
[the remark this answers is not in the document]
Comment (Fernando-Iscar) [reply content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md:127]: Indeed, the picture was not rendering well. 
After my last commit (6385c093), the pictures display correctly.

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md (hunk 1 above)
Comment (Fernando-Iscar) [reply content/building-blocks/automate-and-execute-your-work/reproducible-work/renv.md:177]: ✅Done

In fact, I've accompanied it with the workflow picture, and it looks quite well.

**Comment by Fernando-Iscar** [comment]
All changes are done according to the feedback given. It should be ready to merge @lachlandeer .

**Pull request merged**